### PR TITLE
Pract.04: Add :01-03|Radina Ovcharova

### PR DESCRIPTION
**Pull request by radi676**
Radina Ovcharova
6 3MI0600151

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	{
+		std::cout << "Your number is positive."<<std::endl;
+	}
+	else {
+
+		std::cout << "Your number is not positive." << std::endl;
+	}
+}
+
+void isNegative(int number) {
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:14]: You should return bool

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿
+#include <iostream>
+void isPositive(int number) {
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:3]: bool isPositive

File: SI/Sem.04/Pract.04/Task02.cpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
+﻿
+#include <iostream>
+
+int isEven(int num) {
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task02.cpp:4]: bool isEven

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Some refactoring needed. The functions in task2 and task3 should return bool

**Comment by Justsvetoslavov** [comment]
Closing due to merged solutions.
Address comments and contact us if you have questions.

**Pull request closed without merging**